gravity-restaking/src/token_whitelist.rs

```diff
@@ -35,19 +37,6 @@ pub trait TokenWhitelistModule {
         self.custom_token_decimals(&token_id).clear();
     }
 
-    #[only_owner]
-    #[endpoint(setStakedEgldAmount)]
-    fn set_staked_egld_amount(
-        &self,
-        token_id: TokenIdentifier,
-        staked_egld_for_one_token: BigUint,
-    ) {
-        self.require_token_in_whitelist(&token_id);
-
-        self.staked_egld_for_one_token(&token_id)
-            .set(staked_egld_for_one_token);
-    }
-
```

Comment:
Removed this because it's an unsafe operation, it invalidates the total amounts of staked tokens per user.

Reply:
can you please explain further how it invalidates the total amounts?

Reply:
The math behind the scenes is one staked token equals the given amount. So to calculate the total amount, you do x * staked_egld_amount, which gives you total staked amount. If you ever change staked_egld_amount, this total amount is no longer valid.